### PR TITLE
Added status and language columns

### PR DESCRIPTION
**Pull request by Cheshiriks**
Added status and language columns

Снимок

### CODE REVIEW

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/fossgraph/FossGraphCollectionView.kt [hunk 1]
```diff
@@ -72,6 +73,22 @@ val fossGraphCollectionView: FC<FossGraphCollectionViewProps> = FC { props ->
...
+                        td {
+                            +"${ cellContext.row.original.language }"
+                        }
+                    }
+                }
+                if (!publicTable) {
+                    column(id = "status", header = "Status", { status }) { cellContext ->
+                        Fragment.create {
+                            td {
+                                +"${cellContext.row.original.status}"
```
Comment (sanyavertolet) [inline save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/fossgraph/FossGraphCollectionView.kt:87]: To my mind status should not be upper case

**Review by sanyavertolet — APPROVED** [review]

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/fossgraph/FossGraphCollectionView.kt [hunk 2]
```diff
@@ -140,23 +157,36 @@ val fossGraphCollectionView: FC<FossGraphCollectionViewProps> = FC { props ->
...
-                                            VulnerabilityListTab.OWNER -> VulnerabilityFilter(
-                                                prefixName = "",
-                                                status = VulnerabilityStatus.CREATED,
-                                                isOwner = true,
-                                            )
+                                    when (VulnerabilityListTab.valueOf(value)) {
+                                        VulnerabilityListTab.PUBLIC -> {
+                                            setVulnerabilityFilters {
+                                                VulnerabilityFilter(
+                                                    prefixName = "",
```
Comment (kgevorkyan) [inline save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/fossgraph/FossGraphCollectionView.kt:164]: could it be the default value in `VulnerabilityFilter` itself?

**Review by kgevorkyan — APPROVED** [review]

**Review by nulls — APPROVED** [review]

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/fossgraph/FossGraphCollectionView.kt (hunk 1 above)
Comment (Cheshiriks) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/fossgraph/FossGraphCollectionView.kt:87]: We now display the status in upper case everywhere. I'd leave it like that for the sake of variety.

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/fossgraph/FossGraphCollectionView.kt (hunk 2 above)
Comment (Cheshiriks) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/fossgraph/FossGraphCollectionView.kt]: Changed it

**Pull request merged**